src/Hive.Core/epw_reader/EpwReader.cs

```diff
@@ -0,0 +1,93 @@
+
+using System;
+using System.Collections.Generic;
+using Grasshopper.Kernel.Types;
+using Grasshopper.Kernel;
+using Hive.IO.EnergySystems;
+
+namespace Hive.IO.GhMisc
+{
+    public class EpwReader : GH_Component
+    {
+        public EpwReader()
+          : base("EPW reader", "EpwReader",
+              "Reading .epw weather files, given a local path",
+              "[hive]", "Misc")
+        {
+        }
+
+        public override GH_Exposure Exposure => GH_Exposure.secondary;
+
+
+        protected override void RegisterInputParams(GH_Component.GH_InputParamManager pManager)
+        {
+            pManager.AddTextParameter("Path", "Path", "Path of the epw file", GH_ParamAccess.item);
+        }
```

Comment:
maybe add `pManager.AddBooleanParameter("write GH_Numbers", (...), "additionally to the epw-object, weather file data is output explicitly separately", ...)